<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratePatternEdgeOnSelectedDialog</class>
 <widget class="QDialog" name="GeneratePatternEdgeOnSelectedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frameLibrary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>641</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>51</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>輪郭の幅</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMM1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCancel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="icon">
     <iconset resource="SelectLibraryInMask.qrc">
      <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ドット</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEffective">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Effective Mask</string>
    </property>
    <property name="icon">
     <iconset resource="SelectLibraryInMask.qrc">
      <normaloff>:/Qtres/Effect.png</normaloff>:/Qtres/Effect.png</iconset>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxWidthMM">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxWidthDot">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonNotEffective">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Effective</string>
    </property>
    <property name="icon">
     <iconset resource="SelectLibraryInMask.qrc">
      <normaloff>:/Qtres/EffectNG.png</normaloff>:/Qtres/EffectNG.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>境界分離値</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBoundaryDiff">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="SelectLibraryInMask.qrc"/>
 </resources>
 <connections/>
</ui>
